api.bs

```diff
@@ -727,22 +727,28 @@ The arguments to <a method for=PrivateAttribution>measureConversion()</a> are as
 
 ### Operation ### {#measure-conversion-api-operation}
 
+Given <a dictionary lt=PrivateAttributionConversionOptions>|options|</a>:
+
 1.  Collect the implicit API inputs from the [=relevant settings object=]:
-    1.  The timestamp is set to the [=current high resolution time=].
-    2.  The [=conversion site=] is set to the result of
+    1.  Let |now| be the [=current high resolution time=].
+    1.  Let |topLevelSite| (the [=conversion site=]) be the result of
         [=obtain a site|obtaining a site=] from the [=top-level origin=].
-    3.  The [=intermediary site=] is set to
+    1.  The [=intermediary site=] is set to
         1.   a value of `undefined` if the [=origin=] is [=same site=]
              with the [=top-level origin=],
         1.   otherwise, the result of
              [=obtain a site|obtaining a site=] from the [=origin=].
+        <!-- TODO: the intermediary site is currently unused -->
 1. Validate the page-supplied API inputs:
     1.  If <a dict-member for=PrivateAttributionConversionOptions>logic</a>
         is specified, and the value is anything other than
         <a enum-value for=PrivateAttributionLogic>"last-touch"</a>,
-        return an error.
-1.  If the private attribution API is enabled,
-    invoke the routine to [=do attribution and fill a histogram=].
+        throw a {{TypeError}}.
+    1. If |options|.{{PrivateAttributionConversionOptions/lookbackDays}} is 0,
+        throw a {{RangeError}}.
+1.  If the private attribution API is [[#opt-out|enabled]],
```

Comment:
First let report be the the all-zero histogram. If the api is enabled, set report to the the result of do attribution and fill a histogram.